### PR TITLE
Upgrade dependencies to the latest major release.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,27 +19,22 @@
     "tests"
   ],
   "dependencies": {
-    "alphaville-ui": "Financial-Times/alphaville-ui#^2.6.0",
-    "alphaville-marketslive-chat": "Financial-Times/alphaville-marketslive-chat#^1.0.0",
-    "webchat": "Financial-Times/webchat#^2.1.0",
+    "alphaville-ui": "#c5d9bb6",
```
Comment (notlee) [inline bower.json]: For when [this is released](https://github.com/Financial-Times/alphaville-ui/pull/18):
```suggestion
    "alphaville-ui": "^3.0.0",
```

File: bower.json [hunk 2]
```diff
@@ -19,27 +19,22 @@
...
+    "alphaville-marketslive-chat": "#upgrade-dependencies-25-11-19",
```
Comment (notlee) [inline bower.json]: For when [this is released](https://github.com/Financial-Times/alphaville-marketslive-chat/pull/2):
```suggestion
    "alphaville-marketslive-chat": "^2.0.0",
```

File: bower.json [hunk 3]
```diff
@@ -19,27 +19,22 @@
...
+    "webchat": "#upgrade-dependencies-28-11-19 ",
```
Comment (notlee) [inline bower.json]: For when [this is released](https://github.com/Financial-Times/webchat/pull/2):
```suggestion
    "webchat": "^3.0.0",
```

File: bower.json (hunk 1 above)
Comment (chee) [reply bower.json]: i thought that was a colour at first

File: views/article.handlebars [hunk 4]
```diff
@@ -58,7 +58,7 @@
                     <div class="article__header-inner o-grid-row">
                         <header class="article__header--wrapper " data-o-grid-colspan="12">
                             <div class="article__header-primary">
-                                <h1 class="article__title">{{{article.title}}}</h1>
+                                <h1 class="o-editorial-typography-heading-level-1">{{{article.title}}}</h1>
```
Comment (chee) [inline views/article.handlebars]: nice :)))

File: bower.json (hunk 1 above)
Comment (notlee) [reply bower.json]: #c5d9bb duck shell green

**Pull request merged**